l10n_it_fiscalcode/model/res_partner.py

```diff
@@ -1,12 +1,15 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
-from odoo import models, fields, api
+from codicefiscale import isvalid
+
+from odoo import _, models, fields, api
+from odoo.exceptions import ValidationError
 
 
 class ResPartner(models.Model):
     _inherit = 'res.partner'
 
-    @api.multi
+    @api.constrains('fiscalcode')
```

Comment:
@api.multi non va eliminato

```suggestion
    @api.multi
    @api.constrains('fiscalcode')
```

Reply:
Fatto 👍

Reply:
> Fatto 👍

Grazie.